src/components/dataContainer.js

```diff
@@ -153,9 +153,12 @@
             B.triggerEvent('onNoUserResults');
           }
 
+          if (!data) {
+            return <div />;
+          }
           return (
             <MeProvider value={data} id={model}>
-              <CanvasLayout />
+              {children}
```

Comment:
What do we need this change? `CanvasLayout` is either `BuilderLayout` or `One`, which both render children.